docs/available-filters.adoc

```diff
@@ -4,12 +4,8 @@
 
 Only a few built in filters are provided.
 
-// TODO list them and describe them
+include::available-filters/envelope-encryption.adoc[leveloffset=2]
+include::available-filters/multi-tenancy.adoc[leveloffset=2]
+include::available-filters/schema-validation.adoc[leveloffset=2]
 
-=== Schema-validation
 
-=== Multi-tenancy
-
-// == 3rd party filters
```

Comment:
Do we want to retain an place where we can link to 3rd party filters (and solicit PRs)? Maybe docs aren't the place, but do we have that somewhere?

Reply:
link to https://github.com/kroxylicious/kroxylicious-community-gallery?

Reply:
Done.